composer.json

```diff
@@ -37,7 +37,7 @@
         "laminas/laminas-stdlib": "^3.18",
         "phpbench/phpbench": "^1.2.15",
         "phpunit/phpunit": "^10.5.5",
-        "psalm/plugin-phpunit": "^0.18.4",
+        "psalm/plugin-phpunit": "^0.19.0",
```

Comment:
These changes do not belong in this pull request and should be reverted. Also the `composer.lock` file.